public/tenants/andalusia/config.ts

```diff
@@ -0,0 +1,64 @@
+const config = {
+  tenantName: 'andalusia',
+  tenantURL: 'wald.energizer.de',
```

Comment:
URL and text in this file aren't yet swapped from Energizer :-)

Reply:
currently this branch is broken, need to rewrite the Navbar which can break other branches too